src/pages/Customer.js

```diff
@@ -143,28 +147,46 @@ class CustomerPage extends Component {
   }
 
   handleDocumentNumber = (event) => {
-    const { documentLabel } = this.state
+    const { key } = event
     let document = 'CPF'
-    let value = path(['target', 'value'], event)
+    let value = clean(path(['target', 'value'], event))
+    const documentLength = value.length
+    const hasDocumentReachedLengthLimit = documentLength >= 11
+
+    const pressedNumber = item => item.match(/[0-9]/)
```

Comment:
@allangrds da pra usar o `match` do ramda aqui ;)

```suggestion
    const pressedNumber = match(/[0-9]/)
```